index.html

```diff
@@ -41,23 +53,300 @@ <h3>LET'S CONNECT</h3>
 
       <div class="LinkNetwork">
         <ul>
+          <li>
+            <a href="https://twitter.com/TchasingaJacque"></a>
+            <img src="img/Twitter.png" alt="" />
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/tchasinga-jacques-76aba7214/"
+              ><img src="img/Linkedin icon.png" alt=""
+            /></a>
+          </li>
+          <li>
+            <a href="https://medium.com/@tchasingajacques">
+              <img src="img/Vector.png" alt=""
+            /></a>
+          </li>
+          <li>
+            <a
+              href="https://github.com/tchasinga"
+              target="_blank"
+              rel="noopener noreferrer"
+              ><img src="img/Github.png" alt=""
+            /></a>
+          </li>
+          <li>
+            <a href="https://wellfound.com/jobs"
+              ><img src="img/welldone.png" alt=""
+            /></a>
+          </li>
+        </ul>
+      </div>
+    </main>
+    <!--
+        HERE I'LL USE SESSION TAG TO CODE CARD , AND REMEMBER AGAIN YOU MUST REALLY RUN THIS CODE TO 375PX DEVICE 
+        THANK YOU ..... 
+    -->
+    <section class="card-id">
+      <div class="card-one">
+        <img src="Icon and img/Tonic Portfolio.png" alt="" />
```

Comment:
- [ ] Dear @tchasinga, for your work section, Please make sure to align your contents to the center of the page, Besides that, notice the text for the `See Project` should be centered. This is to make sure you have a design that is identical to the figma design.